src/api/json/catalog.json

```diff
@@ -16,7 +16,7 @@
       "fileMatch": [
         ".aiproj.json"
       ],
-      "url": "https://json.schemastore.org/aiproj"
+      "url": "https://json.schemastore.org/aiproj.json"
```

Comment:
You shouldn't add the file extension

Reply:
As first I tried without but one of the checks failed. Based on this commit, the extension ".json" has become mandatory - https://github.com/SchemaStore/schemastore/commit/748347c6c2d5cfc88096daf74522e8ab4808369c

Reply:
Sorry, you're right. My bad